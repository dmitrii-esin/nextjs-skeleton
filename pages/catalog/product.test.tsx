import { render } from "@testing-library/react";
import Product from "./product";

describe("Product", () => {
    it("renders correctly", () => {
        const { container } = render(<Product />);
        expect(container).toMatchSnapshot();
    });
});
