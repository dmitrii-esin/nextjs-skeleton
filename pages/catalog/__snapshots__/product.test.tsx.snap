// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product renders correctly 1`] = `
<div>
  <hr
    class="MuiDivider-root"
  />
  <div
    class="container"
  >
    Product page
  </div>
</div>
`;
